### ISSUE TITLE
Set speed (0.5x, 1x, 2x) 

### ISSUE THREAD

**Issue by Kowaio** [App2Sales/react-native-hue-player#10]
Hi guys,

First of, thanks a lot for your lib it's a life saver!

I was wondering though how to change the speed of the audio? Do you have any ideas on how to add that?

In RNSound it seems to be possible but it's only for local file but not stream ones.

Thanks again for your help.

**Comment by ammoradi** [comment]
any news?

**Comment by Kowaio** [comment]
Unfortunately no. We changed library with react-native-video that allows this

**Comment by ammoradi** [comment]
@Kowaio how much time this migration did get? does `react-native-video` supports all of this lib's features?

**Comment by Kowaio** [comment]
@ammoradi Yes. Even if it says `video` in the name, it works for streaming audio. And migrating took some time because we wanted special features and control like airplay etc but for the basic I think a couple of days maximum depending on your app complexity.
